HopsanGenerator/templates/fmu3_model.c

```diff
@@ -0,0 +1,320 @@
+#define MODEL_IDENTIFIER fmi3
+
+#include "fmi3Functions.h"
+#include "HopsanEssentials.h"
```

Comment:
Yes, that has always been the case. It is exposing the FMI API functions using `extern "C"`, I think this was the original reason. We can fix it later if it would turn out to be a problem.